pkg/lockfile/parse-pnpm-lock_test.go

```diff
@@ -586,3 +586,13 @@ func TestParsePnpmLock_Files(t *testing.T) {
 		},
 	})
 }
+
+func TestParsePnpmLock_Invalid(t *testing.T) {
```

Comment:
can you assert the packages that are expected to be extracted as part of the test?